tests/acceptance/28_inform_testing/01_files/copy_from02.cf.expected

```diff
@@ -0,0 +1,9 @@
+    info: Created parent directory for '/tmp/TEST.destination'
```

Comment:
```
info: copied /tmp/TEST.destination, depth unlimited, excluding different devices, source /tmp/TEST.source, compare with mtime, checking root directory permissions, keeping timestamp backups, preserve permissions
```

@nickanderson should we use words like "checking, keeping, preserving" or "check, keep, preserve"?
refs:
- https://docs.cfengine.com/docs/3.15/reference-promise-types-files.html#check_root
- https://docs.cfengine.com/docs/3.15/reference-promise-types-files.html#copy_backup